.env

```diff
@@ -0,0 +1,18 @@
+MYSQLDB_USER=root
+MYSQLDB_ROOT_PASSWORD=Danil2704@
+MYSQLDB_DATABASE=task_management
+MYSQLDB_LOCAL_PORT=3308
+MYSQLDB_DOCKER_PORT=3306
+
+SPRING_LOCAL_PORT=8089
+SPRING_DOCKER_PORT=8080
+DEBUG_PORT=5005
+
+JWT_EXPIRATION=300000
+JWT_SECRET=secretdbookdlineonvfcpwepovjpowevjcwopejhvc9
+
+DROPBOX_ACCESS_TOKEN=sl.B6nd-C1EHcU5_luOTAlofoR4invWndkHRormIlqSuphBT6ytPNEq5g6fm3l435ORP00eW5nKLdDTBZ-sLkSrO40jf2T5OlMlPYmQTAFqDxELKSG2sgE1UVzv2xIxrAWhyvF-gCPoVJMRB_wZCGB5iYc
+
+SPRING_MAIL_PORT=587
+SPRING_MAIL_USERNAME=danildryzov@gmail.com
+SPRING_MAIL_PASSWORD=gweeyavnuljuvqde
```

Comment:
let's not expose our .env in the public repo. Instead, you can create .env.sample (for example) with stub values and add a readme instruction to rename this file to .env and change credentials accordingly. Also .env should be added to gitignore